java/src/Ice/src/main/java/com/zeroc/Ice/ObjectAdapter.java

```diff
@@ -4,11 +4,13 @@
 
 import com.zeroc.Ice.Instrumentation.CommunicatorObserver;
 import com.zeroc.Ice.SSL.SSLEngineFactory;
+import java.net.InetAddress;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.Stack;
 import java.util.function.Function;
+import java.util.stream.Collectors;
```

Comment:
You can probably get rid of this now too:
```suggestion
```

Reply:
Fixed.